src/test/resources/interface_credential.sample

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
At some point we need to start using sample data instead of these test files...

Reply:
sure. this file is not in use anymore. so it was deleted